libs/coin-modules/coin-bitcoin/src/wallet-btc/explorer/types.ts

```diff
@@ -6,6 +6,12 @@ export interface IExplorer {
   broadcast(tx: string): Promise<{ data: { result: string } }>;
   getTxHex(txId: string): Promise<string>;
   getFees(): Promise<{ [key: string]: number }>;
+  getNetwork?(): Promise<{
+    relay_fee?: string; // BTC per kB
+    incremental_fee?: string; // BTC per kB
+    version?: string;
+    subversion?: string;
+  }>;
```

Comment:
```suggestion
  getNetwork?(): Promise<NetworkInfoResponse>;
```
Maybe this is not possible idk